shared/message/support/GlobalConfig.proto

```diff
@@ -20,18 +20,8 @@ syntax = "proto3";
 
 package message.support;
 
-/**
- * Field dimensions as defined in the Robocup rules.
- */
 message GlobalConfig {
-
-    /**
-     * A number ID, assigned beginning with 0 or 1.
-     */
+    /// The robots number in the team. Starts at 1. Should be the same as the robots number (nugus1, nugus2, etc)
     uint32 playerId = 1;
-
-    /**
-     * Unique team number.
-     */
-    uint32 teamId = 2;
+    uint32 teamId   = 2;
```

Comment:
Unique team number, assigned by the league (if I am not mistaken)

Reply:
That's good enough for me

Reply:
Wait why isn't this `snake_case` that's a problem...

Reply:
ask and ye shall receive @TrentHouliston #436